build/common.rs

```diff
@@ -17,10 +17,13 @@ pub const V_4_3_2_PATCHES: &[&str] = &[
     "patches/missing_riscv_atomics_fix.diff",
     "patches/missing_xtensa_atomics_fix.diff",
     "patches/pthread_destructor_fix.diff",
+    "patches/esp_app_format_weak.diff",
 ];
 
 #[allow(dead_code)]
-pub const MASTER_PATCHES: &[&str] = &[];
+pub const MASTER_PATCHES: &[&str] = &[
```

Comment:
You'll need to define a `pub const V_4_4_2_PATCHES` and put there the diff. And then use the diff for V4.4.2. And probably the same for `pub const V_5_0_0_PATCHES` (ESP-IDF `master` is V5.1 already)

Reply:
Gotcha